src/views/edit-pet/README.md

```
@@ -0,0 +1,37 @@
+
+| | 온보딩 | 정보 수정 |
+| --- | --- | --- |
+| 헤더 | 닫기 + 진행 표시 | 뒤로가기 + "정보 수정" |
+| 하단 | 이전 / 다음 단계 작성하기 | 수정 완료 하나 |
+| 묶음 | `onbo_002`·`onbo_003`으로 나뉨 | `mypa_121` 한 화면에 모임 |
+
+그래서 단계 컴포넌트를 그대로 쓰지 않고 입력 요소만 재사용한다.
+
+## 라우트
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)